data-analysis/data-tools/data-analysis-using-sql/syntax-ii.md

```diff
@@ -60,9 +60,11 @@ AND title = "%Harry Potter%"
 | 2002 | Harry Potter and the Chamber of Secrets  |
 | 2004 | Harry Potter and the Prisoner of Azkaban |
 
-We can add in multiple `WHERE` clauses using the `AND` keyword. Wrapping "Harry Potter" in `%`, the wildcard character, allows you to pick up all titles that have "Harry Potter" somewhere in the name..
+We can add in multiple `WHERE` clauses using the `AND` keyword. Wrapping "Harry Potter" in `%`, the wildcard character, allows you to pick up all titles that have "Harry Potter" somewhere in the name.
```

Comment:
what do you think about this phrasing?

```suggestion
> 💡 We can add in multiple `WHERE` clauses using the `AND` keyword.

We can use the `LIKE` keyword to look for pattern (instead of exact) matches. The filter `WHERE title LIKE "%Harry Potter%"` tells SQL to look for all the movies that have `"Harry Potter"` somewhere in the name.

This pattern matching is achieved with the `%` wildcard character.
```